index.d.ts

```diff
@@ -1,3 +1,22 @@
+export interface Options {
+	/**
+	Prevent extracted numbers regeneration before all possibilities are exhausted.
+
```

Comment:
> instead include great explanatory docs.

The example helps, but the description needs to be better.

It should in more simple term explain exactly how it works and some common use-cases for it.

Reply:
Just added a better explanation (further the definition) and an use case (cross-fade image carousel).